aten/src/ATen/function_wrapper.py

```diff
@@ -122,7 +122,21 @@ def TypedDict(name, attrs, total=True):  # type: ignore
     ${device_guard_declaration}
 #ifdef HB_REDISPATCH
     if(!${redispatch_condition}) {
-      ${return_call} at::native::${native_type_method_dispatch}(${native_actuals});
+      try {
+        ${return_call} at::native::${native_type_method_dispatch}(${native_actuals});
+      } catch(const c10::Error& c10_error) {
+        if(${fallback_condition} &&
+          (c10_error.msg_without_backtrace().find("missing HammerBlade kernel") != std::string::npos
+           || c10_error.msg_without_backtrace().find("is not implemented for HB, and Fallback is not enabled") != std::string::npos)) {
```

Comment:
Here is one of the hidden conditions -- the fallback-able exception must be HB related -- missing stub or HB backend